### PR TITLE
Create f5-hostsubnet.json

### PR DESCRIPTION
**Pull request by lihongan**
to create hostsubnet for F5 VxLAN tunnel
@bmeng please help review and merge, thanks

### CODE REVIEW

**Comment by bmeng** [comment]
Do you have any other test files which is f5 related? 
If so, I think you should better put all the f5 related test files in an unified sub-folder.

**Comment by lihongan** [comment]
No other special test files for F5 right now. The pod, service, route files are same to those ones for haproxy.

File: networking/f5hostsubnet.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+    "apiVersion": "v1",
+    "metadata": {
+        "name": "f5-server",
+        "annotations": {
+            "pod.network.openshift.io/assign-subnet": "true",
+            "pod.network.openshift.io/fixed-vnid-host": "0"
+           }
+    },
+    "host": "f5-server",
+    "hostIP": "10.20.30.40",
```
Comment (bmeng) [inline networking/f5hostsubnet.json]: Redundant comma.

**Comment by lihongan** [comment]
updated

**Pull request merged**